plone/app/contentlisting/catalog.py

```diff
@@ -39,12 +39,13 @@ def __getattr__(self, name):
 
         if name.startswith('_'):
             raise AttributeError(name)
-        if hasattr(aq_base(self._brain), name):
-            return getattr(self._brain, name)
-        elif hasattr(aq_base(self.getObject()), name):
-            return getattr(aq_base(self.getObject()), name)
-        else:
-            raise AttributeError(name)
+        brain_name = getattr(aq_base(self._brain), name, None)
+        if brain_name is not None:
+            return brain_name
+        object_name = getattr(aq_base(self.getObject()), name, None)
+        if object_name is not None:
+            return object_name
+        raise AttributeError(name)
```

Comment:
@ivanteoh this seems to be the problem of the test failures on jenkins, would you mind reverting this chunk, checking if jenkins is green and merging, and then re-applying this chunk?